### ISSUE TITLE
Attempting to show properties of certain references displays an error about access being denied

### ISSUE THREAD

**Issue by davkean** [dotnet/project-system#1965]
Cannot figure out the logic of which references this occurs to, but if I right-click and choose Properties of System.Drawing here:

![image](https://cloud.githubusercontent.com/assets/1103906/24891793/7563c2a6-1ebb-11e7-9ec5-6ab3947e1166.png)

```
It displays:

---------------------------
Microsoft Visual Studio
---------------------------
Access to the path 'C:\Program Files (x86)\Microsoft Visual Studio\Preview\Enterprise\Common7\IDE\System.Drawing' is denied.
---------------------------
OK   
---------------------------
```

**Comment by davkean** [comment]
Then I get this dialog:
![image](https://cloud.githubusercontent.com/assets/1103906/24891818/a614d5fc-1ebb-11e7-8458-29155e7c07fc.png)

**Comment by davkean** [comment]
The same occurs if I change properties on a reference:

```
An error has occurred while saving the edited properties listed below:
	Copy Local
One or more values are invalid. Access to the path 'C:\Program Files (x86)\Microsoft Visual Studio\Preview\Enterprise\Common7\IDE\System' is denied.

```

**Comment by abpiskunov** [comment]
Where do you see this error? Dialog is dupe of existing issue.

Thanks,
Anton

On Apr 10, 2017, at 8:44 PM, David Kean <notifications@github.com<mailto:notifications@github.com>> wrote:


The same occurs if I change properties on a reference:

An error has occurred while saving the edited properties listed below:
        Copy Local
One or more values are invalid. Access to the path 'C:\Program Files (x86)\Microsoft Visual Studio\Preview\Enterprise\Common7\IDE\System' is denied.

**Comment by davkean** [comment]
Which error? I see both errors changing properties or showing properties window for random references through the ProjectSystem tree.

**Comment by abpiskunov** [comment]
How do you see errors? Dialog or debugger?

Thanks,
Anton

On Apr 10, 2017, at 8:49 PM, David Kean <notifications@github.com<mailto:notifications@github.com>> wrote:


Which error? I see both errors changing properties or showing properties window for random references through the ProjectSystem tree.

**Comment by davkean** [comment]
These are dialogs.

**Comment by abpiskunov** [comment]
This is most likely dupe of https://github.com/dotnet/project-system/issues/397 , although i don't see error dialogs, only properties dialog. Also when i just hit F4 and normal grid opens, changing properties works fine. Will double check tomorrow.

**Comment by davkean** [comment]
No, that is unrelated. This bug isn't about the Properties window opening, this is about the error that is displayed, and note that the Properties window itself is malformed.

**Comment by abpiskunov** [comment]
yes properties window should not be opening at all. But really could not see error dialog - can you attach a screenshot?

**Comment by davkean** [comment]
I've literally CTRL+C the dialog and pasted it above, what value is a screen shot of the dialog going to show you?

**Comment by abpiskunov** [comment]
i see. i just tried it several times , don't see it... thats weird

**Comment by srivatsn** [comment]
Reported through [VSFeedback](https://developercommunity.visualstudio.com/content/problem/40387/error-while-changing-properties-of-depedency-proje.html) as well.

**Comment by srivatsn** [comment]
Fixed by #2000
